.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "root": true,
+  "extends": ["./node_modules/sanctuary-style/eslint-es6.json"],
+  "parserOptions": {
+    "sourceType": "module"
```

Comment:
The only reason I'm providing this linting option is because `test/index.js` is not named as `.mjs` for a reason listed below.